cypress/page-objects/article-preview-page.js

```diff
@@ -0,0 +1,5 @@
+export class ArticlePreviewPage {
+  getTitle () {
+    return cy.get('.article-preview .title')
+  }
+}
```

Comment:
you define this but didn't use it?

Reply:
will use it when the pagination works on cypress, right now I can't select the link I want to open on the preview